app/src/test/java/at/aau/edu/wizards/TooManyPlayerUnitTest.kt

```diff
@@ -0,0 +1,66 @@
+package at.aau.edu.wizards
+
+import at.aau.edu.wizards.api.Server
+import at.aau.edu.wizards.api.model.ServerConnection
+import at.aau.edu.wizards.ui.lobby.LobbyItem
+import at.aau.edu.wizards.ui.lobby.LobbyItemFactory
+import at.aau.edu.wizards.ui.lobby.LobbyViewModel
+import junit.framework.TestCase.assertFalse
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.mock
+
+class TooManyPlayerUnitTest {
```

Comment:
`LobbyViewModelTest`. You write tests for classes present, alway just suffix them with `ClassName...Test`

Reply:
completed